packages/examples/packages/jsx/README.md

```diff
@@ -0,0 +1,7 @@
+# `@metamask/jsx-example-snap`
```

Comment:
If we aren't gonna publish this we should convert at least one of the examples to use JSX for testing purposes

Reply:
I was thinking we do that in a separate PR, and update all examples to use JSX (and maybe add a legacy UI example).

Reply:
We do need at least one example for testing purposes for the integration though